apps/studio/components/interfaces/QueryPerformance/QueryPerformance.tsx

```diff
@@ -108,6 +111,10 @@ export const QueryPerformance = ({
     slowestExecutionTime,
   ])
 
+  useEffect(() => {
+    state.setSelectedDatabaseId(ref)
+  }, [])
```

Comment:
Are you intentionally omitting the `ref` from the deps? If so would you mind adding a comment on why 🙏🏻
```suggestion
  }, [ref])
```

Reply:
good point actually - yeah can add ref in the deps